x-pack/legacy/plugins/siem/public/components/resize_handle/index.tsx

```diff
@@ -21,27 +21,46 @@ export const calculateDeltaX = ({ prevX, screenX }: { prevX: number; screenX: nu
 const isSafari = /^((?!chrome|android|crios|fxios|Firefox).)*safari/i.test(navigator.userAgent);
 
 interface Props {
-  /** the `onResize` callback will be invoked with this id */
-  id: string;
-  /** The resizeable content to render */
-  render: (isResizing: boolean) => React.ReactNode;
+  bottom?: string | number;
   /** a (styled) resize handle */
   handle: React.ReactNode;
   /** optionally provide a height style ResizeHandleContainer */
   height?: string;
+  /** the `onResize` callback will be invoked with this id */
+  id: string;
+  left?: string | number;
   /** invoked when the handle is resized */
   onResize: OnResize;
+  /** The resizeable content to render */
+  position?: string;
+  render: (isResizing: boolean) => React.ReactNode;
+  right?: string | number;
+  top?: string | number;
 }
 
 interface State {
   isResizing: boolean;
 }
 
-const ResizeHandleContainer = styled.div<{ height?: string }>`
-  cursor: ${resizeCursorStyle};
-  ${({ height }) => (height != null ? `height: ${height}` : '')}
+const ResizeHandleContainer = styled.div<{
+  bottom?: string | number;
+  height?: string;
+  left?: string | number;
+  position?: string;
+  right?: string | number;
+  top?: string | number;
+}>`
+  ${({ bottom, height, left, position, right, theme, top }) => css`
+    bottom: ${bottom};
+    cursor: ${resizeCursorStyle};
+    height: ${height};
+    left: ${left};
+    position: ${position};
+    right: ${right};
+    top: ${top};
+    z-index: ${(position === 'absolute' || position === 'relative') && theme.eui.euiZLevel1};
```

Comment:
if either of the `(position === 'absolute' || position === 'relative')` conditions return `false`, the `&&` will short circuit, resulting in an empty value for `z-index`.  Would the following work?

```
    ${position === 'absolute' || position === 'relative'
      ? `z-index: ${theme.eui.euiZLevel1};`
      : ''};
```

Reply:
👍 Good catch! Will make some tweaks to remedy.